sdk/tests/conformance2/sync/sync-webgl-specific.html

```diff
@@ -91,28 +97,23 @@
   // Verify as best as possible that the implementation doesn't allow a sync
   // object to become signaled until returning control to the event loop, by
   // spin-looping for some time.
-  var startTime = Date.now();
-  while (Date.now() - startTime < signaling_retry_ms) {
-    gl.flush();
-    gl.finish();
+  var iter = numRetries;
+  while (--iter > 0) {
+    syncWithGLServer();
     if (gl.getSyncParameter(sync, gl.SYNC_STATUS) == gl.SYNC_SIGNALED) {
       testFailed("Sync object was signaled too early");
       finishTest();
       return;
     }
   }
   testPassed("Sync object wasn't signaled too early");
-  startTimeOfFinish = Date.now();
+  iteration = numRetries;
   requestAnimationFrame(finishSyncParameterTest);
 }
 
-var startTimeOfFinish = 0;
 
 function finishSyncParameterTest() {
-  if (startTimeOfFinish == 0) {
-    startTimeOfFinish = Date.now();
-  }
-  if (Date.now() - startTimeOfFinish > signaling_retry_ms) {
+  if (--iteration == 0) {
     testFailed("Sync object wasn't signaled in a reasonable timeframe (" + (Date.now() - startTimeOfFinish) + " ms)");
```

Comment:
This startTimeOfFinish is no longer defined

Reply:
Thanks for catching this. Fixed.